### PR TITLE
update k8s client libraries to 1.16.4

### PR DESCRIPTION
**Pull request by mhenriks**
Signed-off-by: Michael Henriksen <mhenriks@redhat.com>

**What this PR does / why we need it**:

Time to update libraries!  Currently at 1.13.4.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Update k8s client libraries to 1.16.4
```

### CODE REVIEW

**Comment by mhenriks** [comment]
/hold

Until @rmohr kindly updates the builder image

File: pkg/certificates/triple/cert/cert.go [hunk 1]
```diff
@@ -0,0 +1,269 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```
Comment (rmohr) [inline pkg/certificates/triple/cert/cert.go]: I guess this got removed from the client-go library?

Comment (mhenriks) [reply pkg/certificates/triple/cert/cert.go]: Yup

Comment (rmohr) [reply pkg/certificates/triple/cert/cert.go]: > Yup

/hold cancel

**Comment by rmohr** [comment]
Thanks @mhenriks

**Pull request merged**